root/help.md

```diff
@@ -0,0 +1,41 @@
+% DHCP-CLIENT (1) Container Image Pages
+% Karsten Hopp
```

Comment:
AFAIK, names inside man pages belong to people writing them.

I am not aware of any discussion regarding this topic. Feel free to open a PR on template repo and ask for feedback.